tech/languages/csharp/csharp-ide.md

```diff
@@ -6,27 +6,31 @@ order: 2
 
 # C# IDEs in Fedora
 
+## JetBrains Rider
+
+_.NET Core and Mono_
+
+JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core. If you are familiar with other JetBrains products, such as the IntelliJ or Resharper, you might like this one as well.
+
+[jetbrains.com/rider](http://jetbrains.com/rider)
+
 ## Visual Studio Code
 
 _.NET Core Only_
 
+* VS Code will not work with [our .NET Core packages](/tech/languages/csharp/dotnet-installation.html) due to an [upstream issue](https://github.com/dotnet/source-build/issues/125) affecting OmniSharp. You can still use it with [Microsoft's dotnet packages](https://www.microsoft.com/net/core#linuxfedora).
+
 VS Code is plugin-based IDE that by itself won't just run C#. You have to install the C# plugin, and NuGet Package Manager.
```

Comment:
While seeing this- could you also link the `C# plugin, and NuGet` to their respective install locations?

Reply:
There are several options for the package manager and I've not tested them so I can't recommend. Nothing official... As for the C# plugin, there is one official and a few unofficial, I could link to the official one I guess...